flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointRequestDecider.java

```diff
@@ -188,4 +198,12 @@ int getNumQueuedRequests() {
 		};
 	}
 
+	private static void logInQueueTime(CheckpointTriggerRequest request) {
+		if (LOG.isDebugEnabled()) {
+			long timeInQueue = request.timestamp - currentTimeMillis();
+			if (timeInQueue > LOG_TIME_IN_QUEUE_THRESHOLD_MS) {
+				LOG.debug("checkpoint request time in queue: {}", timeInQueue);
+			}
```

Comment:
bump to `100ms` and log as `INFO`? `DEBUG` wouldn't be visible in the most cases, (for example when users will be reporting to us that checkpoints are not being triggered).